cs-serviceclient-sample/cs-serviceclient-sample/App.xaml

```diff
@@ -18,6 +18,10 @@
              <DataTemplate DataType="{x:Type viewModel:SettingsViewModel}">
                  <view:SettingsView/>
              </DataTemplate>
+             
+             <DataTemplate DataType="{x:Type viewModel:OutputViewModel}">
+                 <view:OutputView/>
```

Comment:
This looks like it spawns a View whereas the bit we got rid of was a `<viewModel:HomeViewModel>`

Reply:
That might be why this bit is ok